src/tutorials/0005/07.md

```diff
@@ -0,0 +1,68 @@
+So far, we've used the CID of the file or directory as the path when accessing a file. However, now that we've learned we can wrap a number of files into a directory, we have a new way to address a file.
+
+As you saw in our lesson on the [wrapWithDirectory option](https://proto.school/#/file-api/05), we can add one or more files to a new directory and, when doing so, we need to provide a name to each one of the files we add. As a result, the `add` method call returns us an array which contains the `hash` value (CID) for each of the files and directories created.
+
+As mentioned earlier, it's useful to think of directories created with `{ wrapWithDirectory: true }` as naming shortcuts, rather than as traditional file folders. Here's why:
+
+Imagine that we called the `add` method like this, in order to add two files wrapped by a root directory...
+
+```javascript
+await ipfs.add([
+    {
+        path: 'lists/kitty-pic-list.txt',
+        content: catList
+    },
+    {
+        path: 'cat-drinking-milk.jpg',
+        content: catPic
+    }
+], { wrapWithDirectory: true })
+```
+
+...and that the `add` method returned this result:
+
+```javascript
+[
+  {
+    "path": "lists/kitty-pic-list.txt",
+    "hash": "Qmey7KyqDwo8BfAoVsyLbybQ8LTN3RGbvvY1zV5PeumTLV",
+    "size": 19021
+  },
+  {
+    "path": "lists",
+    "hash": "QmPT14mWCteuybfrfvqas2L2oin1Y2NCbwzTh9cc33GF1t",
+    "size": 1093102
+  },
+  {
+    "path": "cat-drinking-milk.jpg",
+    "hash": "QmexwNKUeJPmmNR7n4wSzQXrVuyeuQcQikHCHg5xM3mtRq",
+    "size": 912035
+  },
+  {
+    "path": "",
+    "hash": "QmP1j6shbCikCSfnQR7MzJrYdgM6ALpXJAUvkGJFrrwNew",
+    "size": 1181341
+  }
+]
+```
+
+To read the contents of the `kitty-pic-list.txt` file, we would now have three addressing options:
+
+* Using the file's own unique `hash` (CID) as the IPFS path, as we've done previously:
+
+```javascript
+await ipfs.cat("Qmey7KyqDwo8BfAoVsyLbybQ8LTN3RGbvvY1zV5PeumTLV")
+
+```
+* Using the `hash` (CID) of the file's containing directory (`lists`), followed by the file's path relative to that directory:
+
+```javascript
+await ipfs.cat("QmPT14mWCteuybfrfvqas2L2oin1Y2NCbwzTh9cc33GF1t/kitty-pic-list.txt")
```

Comment:
Really this should be prefixed with `/ipfs/`

Reply:
Do you mean it's supposed to be `await ipfs.cat("/ipfs/QmPT14mWCteuybfrfvqas2L2oin1Y2NCbwzTh9cc33GF1t/kitty-pic-list.txt"`)?  Tell me more! 

This is what I see in [the `cat` documentation](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/FILES.md#cat): 
```md
String, including the ipfs handler, a cid and a path to traverse to, ie:
'/ipfs/QmXEmhrMpbVvTh61FNAxP9nU7ygVtyvZA8HZDUaqQCAb66'
'/ipfs/QmXEmhrMpbVvTh61FNAxP9nU7ygVtyvZA8HZDUaqQCAb66/a.txt'
'QmXEmhrMpbVvTh61FNAxP9nU7ygVtyvZA8HZDUaqQCAb66/a.txt'
```

Is there any actual difference between these options? Like does adding /ipfs/ search the whole network and excluding it just searches your node? 

Is this specific to the `cat` command or for all things retrieval-ish, like `get` and `ls` as well?

Reply:
Both work the same (we default to `/ipfs/` if missing), but I agree the tutorial should be explicit to remove any ambiguity:

Without `/ipfs/`  it is not obvious what `QmXEmhrMpbVvTh61FNAxP9nU7ygVtyvZA8HZDUaqQCAb66` is. 
It could be a CID under `/ipfs/`, but it ALSO be a libp2p-key under `/ipns/`.

Reply:
Just adding this note based on a convo with @lidel: We should use `/ipfs/<CID>/<path>` whenever we're using a method that accepts CID/path, so `cat`, `get`, `ls`,  etc. It's only okay to use the CID alone if there is no path after it. In addition to disambiguating from libp2p as mentioned above it will make things more consistent if we later teach `/ipns/` paths. 

I'll accept the suggestion here and then look at the lessons again to see where else this pattern might need to be updated or explained.